addons/website_slides/views/website_slides_templates_profile.xml

```diff
@@ -17,8 +17,14 @@
                     <t t-if="courses_completed" t-call="website_slides.display_course">
                         <t t-set="courses" t-value="courses_completed"></t>
                     </t>
+                    <div t-elif="request.env.user == user" class="text-muted d-inline-block">
```

Comment:
this condition is always True no ?
![screenshot_spect](https://user-images.githubusercontent.com/54316226/145579770-41fefb69-625e-4d01-aad7-d63bd02857d6.png)